Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -57,32 +57,41 @@ In order to configure an external Ceph cluster with Rook, we need to extract som
 - `--restricted-auth-permission`: (optional) Restrict cephCSIKeyrings auth permissions to specific pools, and cluster. Mandatory flags that need to be set are `--rbd-data-pool-name`, and `--cluster-name`. `--cephfs-filesystem-name` flag can also be passed in case of CephFS user restriction, so it can restrict users to particular CephFS filesystem.
 
 !!! note
-    Restricting the csi-users per pool, and per cluster will require creating new csi-users and new secrets for that csi-users, so it can only be used for new deployments.
+    Restricting the csi-users per pool, and per cluster will require creating new csi-users and new secrets for that csi-users.
+    So apply these secrets only to new `Consumer cluster` deployment while using the same `Source cluster`.
 
-    Sample run:
+#### Sample run:
     ```console
     python3 create-external-cluster-resources.py --cephfs-filesystem-name <filesystem-name> --rbd-data-pool-name <pool_name> --cluster-name <cluster-name> --restricted-auth-permission true --format <bash> --rgw-endpoint <rgw_endpoin> --namespace <rook-ceph-external>
     ```
 
-- `--upgrade`: (optional) Upgrades the 'Ceph CSI keyrings (For example: client.csi-cephfs-provisioner) with new permissions needed for the new cluster version and older permission will still be applied.
+#### Multi-tenancy
+        To enable multi-tenancy, run the script with the `--restricted-auth-permission` flag and pass the mandatory flags with it,
+        It will generate the secrets which you can use for creating new `Consumer cluster` deployment using the same `Source cluster`(ceph cluster).
+        So you would be running different isolated consumer clusters on top of single `Source cluster`.
 
-    Sample run:
+#### Upgrades
+- `--upgrade`: (optional) Upgrades the 'Ceph CSI keyrings (For example: client.csi-cephfs-provisioner) with new permissions needed for the new cluster version and older permission will still be applied.
```

Comment:
This --upgrade flag would be best moved above with all the other flags. Then the sample run just stays here.